CONTRIBUTING.md

```diff
@@ -42,8 +44,90 @@ mvn -f pom.client.xml -Dgpg.skip -DskipTests clean install
 >**Note**: Refer to [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Building) for learning about how to build using Java 11
 
 ### Compiling one project only
+
 ```
 mvn -f sdk/{projectForlderDir}/pom.xml -Dgpg.skip clean install
 
 //example: mvn -f sdk/keyvault/azure-keyvault-keys/pom.xml clean install
 ```
+
+## Versions and versioning
+
+Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there version text files, one for client (version_client.txt) and one for data (version_data.txt). The format of the version files is as follows:
```

Comment:
"directory there version text files" is missing an "exists" or "are"

Reply:
Will be updated as part of https://github.com/Azure/azure-sdk-for-java/pull/5889